docs/BREAKINGCHANGES.md

```diff
@@ -6,151 +8,125 @@ TODO
 TODO
 
 
-### Add error on Legacy Credential over non-HTTPS for Web Cmdlets [5402](https://github.com/PowerShell/PowerShell/issues/5402)
-TODO
-
-
 ### Change $OutputEncoding to use iso-8859-1 encoding rather than ASCII [5361](https://github.com/PowerShell/PowerShell/issues/5361)
-TODO
-
+This is marked as a breaking change because the output of the pipeline is not altered as it was previously, anyone relying on ascii encoding (especially for extended ascii sequences) will no longer have that behavior.
 
-### Fix Single Value JSON null in Invoke-RestMethod [5338](https://github.com/PowerShell/PowerShell/issues/5338)
-TODO
+TODO: This is closed in favor of a new PR which simply changes the output encoding to utf8nobom, the new PR is not referenced.  Need to find and see if it is still a breaking change.
 
 
 ### Invoke-RestMethod doesn't return useful info when no data is returned. [5320](https://github.com/PowerShell/PowerShell/issues/5320)
-TODO
+When an API returns just `null`, Invoke-RestMethod was serializing this as the string `"null"` instead of `$null`.  This change fixes the logic in `Invoke-RestMethod` to properly serialize a valid single value JSON `null` literal as `$null`.
 
 
 ### Remove DCOM support from *-Computer cmdlets (replacement for 5152) [5277](https://github.com/PowerShell/PowerShell/issues/5277)
-TODO
+DCOM is not supported in corefx, therefore the *-Computer cmdlets were removed.
 
 
 ### Remove AllScope from most default aliases [5268](https://github.com/PowerShell/PowerShell/issues/5268)
-TODO
-
-
-### Fix performance issues in Add-Type [5243](https://github.com/PowerShell/PowerShell/issues/5243)
-TODO
+To speed up spope creation, AllScope was removed from most default aliases.  AllScope was left for a few frequently used aliases where the lookup was faster.
 
 
 ### ConvertFrom-Json silently throws away duplicated keys with identical casing [5199](https://github.com/PowerShell/PowerShell/issues/5199)
-TODO
+TODO: Still open
 
 
 ### fix get-item -literalpath a*b if `a*b` doesn't actually exist to return error [5197](https://github.com/PowerShell/PowerShell/issues/5197)
-TODO
+Previously, `-literalpath` given a wildcard will treat it the same as -path and if the wildcard found no files, it would silently exit. Correct behavior should be that -literalpath is literal so if the file doesn't exist, it should error. Fix is to also see if wildcard is being suppressed.
```

Comment:
Last sentence should be: Change is to treat wildcards used with `-Literal` as literal.

Also, use semantic linefeeds.